apm-configuration/deno.mdx

```diff
@@ -80,7 +80,68 @@ error("error");
 debug("debug");
 ```
 
+# Hono Framework 
```

Comment:
@eduardo-mw let's add some comments in this code to give small descriptions on what each important block does

Reply:
@Zamichel added. Lmk what you think